styles.css

```diff
@@ -0,0 +1,183 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@900&display=swap");
+
+body {
+  background-image: url(./images/Header\ bg@2x.png);
+  background-position: center;
+  background-size: contain;
+  background-repeat: no-repeat;
+  padding: 0 5%;
+}
```

Comment:
### Optional 

- [ ] Kindly add background image to the appropriate section and readjust your [background position](https://www.w3schools.com/cssref/pr_background-position.asp)